main.go

```diff
@@ -13,16 +12,24 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+
+	"github.com/dave/dst"
+	"github.com/dave/dst/decorator"
 )
 
-const godocCommentFormat = "// %s missing godoc"
+var godocCommentFormat = "// %s missing godoc"
+var godocAllowNoSpace = false
 
 func main() {
 	wd, err := os.Getwd()
 	if err != nil {
 		log.Fatalf("error getting current working directory: %v", err)
 	}
-
+	commentFormat := flag.String("format", "// %s missing godoc", "comment format")
+	allowNoSpace := flag.Bool("allow_no_space", false, "allow comments without space between // and type name")
+	flag.Parse()
+	godocCommentFormat = *commentFormat
+	godocAllowNoSpace = *allowNoSpace
```

Comment:
Can you extract those in a init function as it is the idiomatic way to handle flags?

Reply:
Also, in the `commentFormat` flag, use the corresponding constant as a default value.

Reply:
`--allow_no_space` is not so intuitive as it uses underscores. Please use `--allow-no-space` dash notation.

Reply:
done